cmd/soroban-cli/src/commands/tx/new/manage_data.rs

```diff
@@ -0,0 +1,45 @@
+use clap::{command, Parser};
+
+use crate::{
+    commands::{global, tx},
+    tx::builder,
+    xdr,
+};
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub tx: tx::args::Args,
+    /// Line to change, either 4 or 12 alphanumeric characters, or "native" if not specified
+    #[arg(long)]
+    pub data_name: xdr::StringM<64>,
+    /// Up to 64 bytes long hex string
+    /// If not present then the existing Name will be deleted.
+    /// If present then this value will be set in the `DataEntry`.
+    #[arg(long)]
+    pub data_value: Option<xdr::BytesM<64>>,
```

Comment:
That does seem a little more elegant, but balancing the impact / benefit with maintaining, and setting user expectations, it may also be beneficial if users can rely on these tx building commands to reflect the networks structure/design for the operations, so that devs don't have to remember / reconcile the differences between the network and the CLI interface.

For e.g. if I read stellar docs I learn that a manage-data op with key and no value deletes the key. Then I use the CLI, but now I learn that there's two different manage-data ops, an add and a delete, but then I look at an SDK, and I don't find an add and delete op. Confusion.

Vs I read stellar docs I learn that a manage-data op with key and no value deletes the key. Then I use the CLI, and I do same, a manage-data op with key and no value when I need to delete. Then I look at an SDK, and I do same, a manage-data op with key and no value when I need to delete.

Reply:
Hmm, agree, when coming from this angle it's better to align the tooling we have